terminal.html

```diff
@@ -1,317 +0,0 @@
-<html>
```

Comment:
The link in the index file links to this file, I'd prefer if we could keep this file, since terminalT was made mostly just to make testing the typer easier.